Documentation/Configuration/Typo3ConfVars/GFX.rst

```diff
@@ -183,16 +183,53 @@ processor_stripColorProfileByDefault
 processor_stripColorProfileCommand
 ==================================
 
-.. confval:: $GLOBALS['TYPO3_CONF_VARS']['GFX']['processor_stripColorProfileCommand']
+..  versionchanged:: 11.5.35/12.4.11
+    This string-based configuration option has been superseded by
+    :ref:`$GLOBALS['TYPO3_CONF_VARS']['GFX']['processor_stripColorProfileParameters'] <typo3ConfVars_gfx_processor_stripColorProfileParameters>`
+    for `security reasons <https://typo3.org/security/advisory/typo3-core-sa-2024-002>`__.
 
-   :type: text
-   :Default: \+profile \'\*\'
+..  confval:: $GLOBALS['TYPO3_CONF_VARS']['GFX']['processor_stripColorProfileCommand']
+
+    :type: string
+
+    This option expected a string of command line parameters. The defined
+    parameters had to be shell-escaped beforehand, while the new option
+    :ref:`typo3ConfVars_gfx_processor_stripColorProfileParameters` expects
+    an array of strings that will be shell-escaped by TYPO3 when used.
+
+    The existing configuration will continue to be supported. Still, it is
+    suggested to use the new configuration format, as the Install Tool is
```

Comment:
`:guilabel:` here for "Install Tool"?

Reply:
Ah, so it only applies to "real" modules and not the "stand-alone" module that the Install Tool is. Understood, thanks.

Reply:
Yup, the guilabel should show, that there is something on the GUI.